README.md

```diff
@@ -70,6 +70,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 *Tools that provide metrics and quality measurements.*
 
 * [Checkstyle](http://checkstyle.sourceforge.net/) - Static analysis of coding conventions and standards.
+* [ErrorProne](https://github.com/google/error-prone) - Catch common Java mistakes as compile-time errors from Google.
```

Comment:
The description is kind of misleading. You wanted to say that Google made the library but it sounds like it catches "Java  errors coming from Google"